sections/quick-order-form.liquid

```diff
@@ -67,13 +99,15 @@
             </thead>
 
             <tbody>
-              {%- render 'quick-order-list-row',
-                item: section.settings.collection.products.first,
-                image: section.settings.collection.products.first.featured_media,
-                sku: section.settings.collection.products.first.selected_or_first_available_variant.sku,
-                variant: section.settings.collection.products.first.selected_or_first_available_variant
+              {%-
+                render 'quick-order-list-row',
+                item: collections['shop-all'].products.first,
+                image: collections['shop-all']products.first.featured_media,
+                sku: collections['shop-all']products.first.selected_or_first_available_variant.sku,
+                variant: collections['shop-all']products.first.selected_or_first_available_variant
               -%}
-              {%- for variant in section.settings.collection.products.first.variants -%}
+              <tr class="empty-row"></tr>
```

Comment:
Why do we need an empty row?

Reply:
I don't like this solution much either; however, I couldn't find a better way to achieve the desired design. According to Figma, there is a space between the product and variant elements. 
<img width="810" alt="Screenshot 2023-11-02 at 1 15 57 PM" src="https://github.com/Shopify/dawn/assets/105315663/bb2ed368-90fd-4ec6-b9f4-cf66b7ed7343">

We can add the separator line by applying a `box-shadow` to the product element. However, there is no way to add space after that line. Since we have a `table` with `display: table`, we cannot use margin. We can only use `padding` for elements inside the table. Unfortunately, applying `padding` to variant elements is not working for us because we have a `hover` effect. Let me know what you think or if you have any workarounds.

As an alternative, we can simply remove that space, and it will look like this.
<img width="895" alt="Screenshot 2023-11-02 at 1 13 40 PM" src="https://github.com/Shopify/dawn/assets/105315663/b5502c85-1988-44db-bcbc-d146115136f5">

Reply:
The alternative looks good to me @eugenekasimov without the space. 👍

Reply:
Done here [835d2a9](https://github.com/Shopify/dawn/pull/3072/commits/835d2a9e4846eb23eb34a3e656a357da0bf6724e)